Gitops process (#27)
giantswarm/handbook: CHANGELOG.md

## [Unreleased]

### Added

> - Add a ops recipe to trouble shoot GitOps environments.

Reviewer: ```suggestion
- Add an ops recipe to troubleshoot GitOps environments
```
